authorize/evaluator/policy_evaluator.go

```diff
@@ -147,7 +147,8 @@ func NewPolicyEvaluator(
 
 	// for each script, create a rego and prepare a query.
 	for i := range e.queries {
-		log.Ctx(ctx).Debug().
+		log.Ctx(ctx).
+			Trace().
```

Comment:
This is an intentional change from debug to trace? Is the `script` field a bunch of Rego, and generally just extra noise in the logs?

Reply:
This happened to be cherry-picked along with changes to the log.Ctx line, but yes this writes out massive rego strings and we should probably change it.

Reply:
Sounds good, just wanted to double check.